test/test_datasets.py

```diff
@@ -2205,5 +2205,49 @@ def inject_fake_data(self, tmpdir: str, config):
         return len(sampled_classes * n_samples_per_class)
 
 
+class Country211TestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.Country211
+    FEATURE_TYPES = (PIL.Image.Image, int)
```

Comment:
This is the default for `ImageDatasetTestCase` so we don't need to set it at all.